views/administration/users_edit.hbs

```diff
@@ -61,82 +61,96 @@
+                        <input name="parent_consentId" type="hidden" value="{{@root.consent.parentConsent._id}}" disabled />
+
+                        <label style="margin: 15px 10px 10px 0"><input name="parent_form" value="analog" type="radio"
+                                disabled {{#ifeq @root.consent.parentConsent.form 'analog'}}checked{{/ifeq}} />
+                            {{$t "administration.label.analog" }}</label>
+                        <label><input name="parent_form" value="digital" type="radio" disabled
+                                {{#ifeq @root.consent.parentConsent.form 'digital'}}checked{{/ifeq}} />
+                            {{$t "administration.label.digital" }}</label>
+                        <br />
+                        <label><input name="parent_privacyConsent" type="checkbox" value="true" disabled
+                                {{#if @root.consent.parentConsent.privacyConsent}}checked{{/if}} />
+                            {{$t "administration.global.label.dataProtection" }}</label><br />
+                        <label style="margin-bottom: 20px"><input name="parent_termsOfUseConsent" type="checkbox"
+                                value="true" disabled {{#if @root.consent.parentConsent.termsOfUseConsent}}checked{{/if}} />
+                            {{$t "administration.label.termsOfUse" }}</label><br />
+                    </div>
+                    <div class="col-md-6">
+                        <b>{{$t "administration.global.label.student" }}</b><br />
+                        <input name="student_consentId" type="hidden" value="{{@root.consent._id}}" disabled />
+
+                        <label style="margin: 15px 10px 10px 0"><input name="student_form" value="analog" type="radio"
+                                disabled {{#ifeq @root.consent.userConsent.form 'analog'}}checked{{/ifeq}} />
+                            {{$t "administration.label.analog" }}</label>
+                        <label><input name="student_form" value="digital" type="radio" disabled
+                                {{#ifeq @root.consent.userConsent.form 'digital'}}checked{{/ifeq}} />
+                            {{$t "administration.label.digital" }}</label>
+                        <br />
+                        <label><input name="student_privacyConsent" type="checkbox" value="true" disabled
+                                {{#if @root.consent.userConsent.privacyConsent}}checked{{/if}} />
+                            {{$t "administration.global.label.dataProtection" }}</label><br />
+                        <label style="margin-bottom: 20px"><input name="student_termsOfUseConsent" type="checkbox"
+                                value="true" disabled {{#if @root.consent.userConsent.termsOfUseConsent}}checked{{/if}} />
+                            {{$t "administration.label.termsOfUse" }}</label><br />
+                    </div>
+                    {{else}}
```

Comment:
so indentation comment above, thats why it is unclear here for example to which condition this else belongs